Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -172,24 +160,24 @@ def get_indicators_command(client: Client, args: Dict[str, str]) -> Tuple[str, D
             values = item.get(indicator_type_lower)
```

Comment:
This command `get_indicators_command` is practically a duplicate of the `fetch_indicators_command`. Why not simply call the `fetch_indicators_command` from this one. You should also provide a `limit` argument to the `get_indicators_command` which can be passed to the `fetch_indicators_command`.